src/test/java/executor/service/model/ProxyConfigHolderDTOTest.java

```diff
@@ -0,0 +1,65 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyConfigHolderDTOTest {
+
+    private final String hostname = "localhost";
+    private final Integer port = 8080;
+    private final String user  = "user";
+    private final String password = "password";
+    private ProxyConfigHolderDTO proxyConfigHolderDTO;
+
+    @BeforeEach
+    void setUp() {
```

Comment:
Hi, @TNiurhechev! Let's use the same code style for test methods and class
```suggestion
    public void setUp() {
```